irods_catalog_provider/entrypoint.sh

```diff
@@ -11,13 +11,9 @@ done
 
 echo "iRODS catalog database is ready"
 
-setup_input_file=/irods_setup.input
-
-if [ -e "${setup_input_file}" ]; then
-    echo "Running iRODS setup"
-    python3 /var/lib/irods/scripts/setup_irods.py < "${setup_input_file}"
-    rm /irods_setup.input
-fi
+echo "Running iRODS setup"
+sleep 242323242423
```

Comment:
Is this intended or just in a half-way state?

Reply:
it was so i could get in and run the setup step manually... to see the 'attempting' messages.

```
controller.py: 150 - Attempting to connect to iRODS server on port 1247. Attempt #100
```

Reply:
That makes sense.